beacon-chain/rpc/prysm/v1alpha1/validator/proposer.go

```diff
@@ -495,43 +495,22 @@ func (vs *Server) broadcastAndReceiveBlobs(ctx context.Context, sidecars []*ethp
 }
 
 // broadcastAndReceiveDataColumns handles the broadcasting and reception of data columns sidecars.
-func (vs *Server) broadcastAndReceiveDataColumns(
-	ctx context.Context,
-	roSidecars []blocks.RODataColumn,
-	root [fieldparams.RootLength]byte,
```

Comment:
was this not even used by the function?

Reply:
I think it was used in the past, then we changed the function.
And because `unused` is not activated by the linter, nobody noticed!